<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 149, 151);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imagen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>EXpx4zXUMAAtJja.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imagen_2">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>10</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/Logos/FIT_logo_vertical.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imagen_3">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>650</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/Ejercicios/ImagenGatoEuropeo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_iniciar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; color: black;</string>
    </property>
    <property name="text">
     <string>CONECTAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_estado">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ESTADO</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_datos">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; color: black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_datos">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Datos</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_datos_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DHT11/Transistor</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_datos_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>470</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; color: black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_datos_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>430</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Datos</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_datos_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fotoreceptor</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_datos_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>470</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; color: black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_datos_5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>380</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sensor Ultrasonico</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_datos_6">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>430</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Datos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mostrar_1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; color: black;</string>
    </property>
    <property name="text">
     <string>MOSTRAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mostrar_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>430</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; color: black;</string>
    </property>
    <property name="text">
     <string>MOSTRAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mostrar_3">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>430</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; color: black;</string>
    </property>
    <property name="text">
     <string>MOSTRAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>-30</y>
      <width>861</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; vertical-align:sub;&quot;&gt;SISTEMA DE ADMINISTRACIÃ“N PARA AUTOMOVIL SENSORIAL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>imagen</zorder>
   <zorder>imagen_2</zorder>
   <zorder>imagen_3</zorder>
   <zorder>txt_estado</zorder>
   <zorder>lista_datos</zorder>
   <zorder>txt_datos</zorder>
   <zorder>txt_datos_2</zorder>
   <zorder>lista_datos_2</zorder>
   <zorder>txt_datos_3</zorder>
   <zorder>txt_datos_4</zorder>
   <zorder>lista_datos_3</zorder>
   <zorder>txt_datos_5</zorder>
   <zorder>txt_datos_6</zorder>
   <zorder>btn_mostrar_1</zorder>
   <zorder>btn_mostrar_2</zorder>
   <zorder>btn_mostrar_3</zorder>
   <zorder>label_2</zorder>
   <zorder>btn_iniciar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
